notes/CodeFlag_notes.csv

```diff
@@ -138,3 +138,5 @@ noteID,note
 141,"See Appendix XYZ."
 142,"In the ring ordering scheme, the 12 rhomboids are not considered and the scanning mode applies to the grid in its entirety."
 143,"In the nested ordering scheme, the 12 rhomboids are scanned in order defined in Gorsky et al. 2015. The flag bits then refer to how the points are scanned within a rhomboid."
+221a,"For code number 102 the drought index encoded in the message is specified by the table entries from discipline 0 and parameter category 22 with the corresponding parameter number. The statistical processing period of the drought index is defined by the keys ""length of time range"" and ""indicator of unit of time for the increment"""
+221b,"WMO proposed indices (https://library.wmo.int/doc_num.php?explnum_id=3057). The index is specified by setting discipline and parameter category / number to the specific index to be encoded. Drought indicators are calculated over different time ranges, prescribed by the keys lengthOfTimeRange and indicatorOfUnitForTimeRange in the templates 4.107, 4.108, 4.109 and 4.112. All standardised drought indices are calculated according to WMO guidelines of the SPI (https://library.wmo.int/doc_num.php?explnum_id=7768)."
```

Comment:
update pub URL with the permanent link

Reply:
@sebvi -- this doesn't make any sense as a sentence. 

"WMO proposed indices (https://library.wmo.int/doc_num.php?explnum_id=3057)."